lib/unleash/client.rb

```diff
@@ -53,6 +62,27 @@ def is_enabled?(feature, context = nil, default_value = false)
         return toggle_result
     end
 
+    # enabled? is a more ruby idiomatic method name than is_enabled?
+    alias_method :enabled?, :is_enabled?
+
+
```

Comment:
consider a `flush`/`flush_statistics` method, which does not do the `shutdown!` at the end.